<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CRM - Sistema de Gestion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* === GLOBAL === */
QMainWindow {
    background-color: #f0f2f5;
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
}

/* === SIDEBAR === */
#sidebar {
    background-color: #1a1a2e;
    min-width: 260px;
    max-width: 260px;
}
#sidebarTitle {
    color: #ffffff;
    font-size: 20px;
    font-weight: bold;
    padding: 10px 0px;
}
#sidebarSubtitle {
    color: #7f8c9b;
    font-size: 12px;
    padding-bottom: 5px;
}
#sidebarSeparator {
    background-color: #2d2d4a;
    min-height: 1px;
    max-height: 1px;
}

/* Sidebar buttons base */
#btnDashboard, #btnClientes, #btnProductos,
#btnVentas, #btnReportes, #btnUsuarios, #btnConfiguracion {
    background-color: transparent;
    color: #a0aec0;
    font-size: 15px;
    font-weight: 500;
    text-align: left;
    padding: 14px 20px;
    border: none;
    border-radius: 8px;
    margin: 2px 12px;
}
#btnDashboard:hover, #btnClientes:hover, #btnProductos:hover,
#btnVentas:hover, #btnReportes:hover, #btnUsuarios:hover, #btnConfiguracion:hover {
    background-color: #16213e;
    color: #ffffff;
}

/* Active state for dashboard (default) */
#btnDashboard {
    background-color: #4a90d9;
    color: #ffffff;
    font-weight: 600;
}

#btnCerrarSesion {
    background-color: transparent;
    color: #e74c5e;
    font-size: 14px;
    font-weight: 500;
    text-align: left;
    padding: 14px 20px;
    border: 1px solid #3d2030;
    border-radius: 8px;
    margin: 2px 12px;
}
#btnCerrarSesion:hover {
    background-color: #3d2030;
    color: #ff6b7a;
}

/* === HEADER === */
#header {
    background-color: #ffffff;
    border-bottom: 1px solid #e2e8f0;
}
#headerPageTitle {
    color: #1a1a2e;
    font-size: 24px;
    font-weight: bold;
}
#headerUserName {
    color: #1a1a2e;
    font-size: 14px;
    font-weight: 600;
}
#headerUserRole {
    color: #7f8c9b;
    font-size: 12px;
}

/* === CONTENT === */
#contentArea {
    background-color: #f0f2f5;
}
#welcomeTitle {
    color: #1a1a2e;
    font-size: 32px;
    font-weight: bold;
}
#welcomeSubtitle {
    color: #7f8c9b;
    font-size: 16px;
}

/* Stat cards */
#cardClientes, #cardProductos, #cardVentas, #cardReportes {
    background-color: #ffffff;
    border-radius: 12px;
    border: 1px solid #e2e8f0;
}
#cardClientes:hover, #cardProductos:hover, #cardVentas:hover, #cardReportes:hover {
    border: 1px solid #4a90d9;
}

#cardClientesIcon, #cardProductosIcon, #cardVentasIcon, #cardReportesIcon {
    font-size: 36px;
}
#cardClientesTitle, #cardProductosTitle, #cardVentasTitle, #cardReportesTitle {
    color: #1a1a2e;
    font-size: 18px;
    font-weight: bold;
}
#cardClientesDesc, #cardProductosDesc, #cardVentasDesc, #cardReportesDesc {
    color: #7f8c9b;
    font-size: 13px;
}

/* Footer */
#footerLabel {
    color: #a0aec0;
    font-size: 12px;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="rootLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <layout class="QVBoxLayout" name="sidebarLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="sidebarTitle">
         <property name="text">
          <string>   CRM System</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sidebarSubtitle">
         <property name="text">
          <string>   Sistema de Gestion Empresarial</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sidebarPreSep">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="sidebarSeparator" native="true"/>
       </item>
       <item>
        <spacer name="sidebarPostSep">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClientes">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Clientes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnProductos">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Productos</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnVentas">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Ventas</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnReportes">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Reportes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnUsuarios">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Usuarios</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnConfiguracion">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Configuracion</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sidebarBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="sidebarSeparator2" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: #2d2d4a; min-height: 1px; max-height: 1px;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sidebarPreLogout">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCerrarSesion">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>  Cerrar Sesion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightPanel" native="true">
      <layout class="QVBoxLayout" name="rightPanelLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="header" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="headerLayout">
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="headerPageTitle">
            <property name="text">
             <string>Dashboard</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="headerSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="headerUserInfoLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="headerUserName">
              <property name="text">
               <string>Usuario</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="headerUserRole">
              <property name="text">
               <string>Administrador</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="contentArea" native="true">
         <layout class="QVBoxLayout" name="contentLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="welcomeTitle">
            <property name="text">
             <string>Bienvenido al Sistema</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="welcomeSubtitle">
            <property name="text">
             <string>Selecciona una opcion del menu para comenzar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="contentPreCards">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="cardsRow">
            <property name="spacing">
             <number>20</number>
            </property>
           </layout>
          </item>
          <item>
           <spacer name="contentBottomSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="footerLabel">
            <property name="text">
             <string>CRM - Sistema de Gestion Empresarial v1.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
